### PR TITLE
organize contracts setup in network test file

### PR DESCRIPTION
**Pull request by ilanDoron**

### CODE REVIEW

File: contractsSol5/KyberNetwork.sol [hunk 1]
```diff
@@ -47,8 +47,8 @@ contract KyberNetwork is WithdrawableNoModifiers, Utils4, IKyberNetwork, Reentra
     mapping(address=>bool) internal kyberProxyContracts;
 
     // mapping reserve ID to address, keeps an array of all previous reserve addresses with this ID
-    mapping(bytes32=>address) public reserveIdToAddress;
-    mapping(address=>address) public reserveRebateWallet;
+    mapping(bytes32=>address) internal reserveIdToAddress;
+    mapping(address=>address) internal reserveRebateWallet;
```
Comment (Anyhowclick) [inline contractsSol5/KyberNetwork.sol:51]: how will reserves check their rebate wallets?

Comment (ilanDoron) [reply contractsSol5/KyberNetwork.sol:51]: right
will revert this change for rebate wallet.

Comment (ilanDoron) [reply contractsSol5/KyberNetwork.sol]: fixed

File: test/sol5/kyberNetwork.js [hunk 2]
```diff
@@ -319,13 +325,21 @@ contract('KyberNetwork', function(accounts) {
         });
 
         it("add a few matchingEngine + feeHandler contracts, see event + updated in getter.", async() => {
-            let txResult = await tempNetwork.setContracts(handler1, matchingEngine1, storage1, zeroAddress, {from: admin});
+            tempStorage2 = await KyberStorage.new(admin);
+            tempMatchingEngine1 = MatchingEngine.new(admin);
+            tempStorage.setNetworkContract(tempNetwork.address, {from: admin});
```
Comment (manhlx3006) [inline test/sol5/kyberNetwork.js:330]: need `await` here, 2 lines below as well

**Pull request merged**